source/api/discovery/1/context.json

```diff
@@ -0,0 +1,183 @@
+{
+  "@context": [
+  	"https://www.w3.org/ns/activitystreams",
+  	{
+      "as": "https://www.w3.org/ns/activitystreams#",
+      "dcterms": "http://purl.org/dc/terms/",
+      "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+      "discov": "https://iiif.io/api/discovery/1#",
+      "oa": "http://www.w3.org/ns/oa#",
+
+      "OrderedCollection": {
+        "@id": "as:OrderedCollection",
+        "@context": {
+          "rights": {
+            "@id": "dcterms:rights",
+            "@type": "@id"
+          },
+          "partOf": {
+            "@id": "dcterms:partOf",
+            "@type": "@id"
+          }
+        }
+      },
+
+      "Refresh": {
+        "@id": "discov:Refresh",
+        "@context": {
+          "partOf": {
+            "@id": "dcterms:partOf",
+            "@type": "@id"
+          }
+        }        
+      },
+
+      "Create": {
+        "@id": "as:Create",
+        "@context": {
+          "partOf": {
+            "@id": "dcterms:partOf",
+            "@type": "@id"
+          }
+        }
+      },
+
+      "Update": {
+        "@id": "as:Update",
+        "@context": {
+          "partOf": {
+            "@id": "dcterms:partOf",
+            "@type": "@id"
+          }
+        }
+      },
+
+      "Delete": {
+        "@id": "as:Delete",
+        "@context": {
+          "partOf": {
+            "@id": "dcterms:partOf",
+            "@type": "@id"
+          }
+        }
+      },
+
+      "Add": {
+        "@id": "as:Delete",
+        "@context": {
+          "partOf": {
+            "@id": "dcterms:partOf",
+            "@type": "@id"
+          }
+        }
+      },
+
+      "Remove": {
+        "@id": "as:Delete",
+        "@context": {
+          "partOf": {
+            "@id": "dcterms:partOf",
+            "@type": "@id"
+          }
+        }
+      },
+      "Move": {
+        "@id": "as:Delete",
+        "@context": {
+          "partOf": {
+            "@id": "dcterms:partOf",
+            "@type": "@id"
+          }
+        }
+      },
+
+
+      "label": {
+        "@id": "rdfs:label",
+        "@container": ["@language", "@set"],
+        "@context": {
+          "none": "@none"
+        }
+      },    
+      "seeAlso": {
+        "@type": "@id",
+        "@id": "rdfs:seeAlso",
+        "@container": "@set"
+      },
+
+      "format": {
+        "@id": "dc:format"
+      },
+      "profile": {
+        "@type": "@vocab",
+        "@id": "dcterms:conformsTo"
+      },
+      "canonical": {
+        "@type": "@id",
+        "@id": "oa:canonical"
+      },
+      "provider": {
+        "@type": "@id",
+        "@id": "schema:provider",
```

Comment:
schema.org not imported at top of context?

Reply:
Fixed, good catch!